pkg/policy/node.go

```diff
@@ -26,11 +26,16 @@ import (
 
 // Node to define hierarchy of rules
 type Node struct {
-	path      string
-	Name      string           `json:"name"`
-	Parent    *Node            `json:"-"`
-	Rules     []PolicyRule     `json:"rules,omitempty"`
-	Children  map[string]*Node `json:"children,omitempty"`
+	path     string
+	Name     string           `json:"name"`
+	Parent   *Node            `json:"-"`
+	Rules    []PolicyRule     `json:"rules,omitempty"`
+	Children map[string]*Node `json:"children,omitempty"`
+
+	// CoverAll marks policy nodes which should always have their
+	// rules applied regardless of whether the name/path of the node
+	// matches any of the labels in the search context.
+	CoverAll  bool `json:"coverAll,omitempty"`
```

Comment:
With coverAll=true this test fails but it should pass.
```
	ctx = policy.SearchContext{
		From: labels.LabelArray{
			labels.NewLabel("foo4", "bar4", common.K8sLabelSource),
			labels.NewLabel("foo3", "bar3", common.K8sLabelSource),
		},
		To: labels.LabelArray{
			labels.NewLabel("foo2", "bar2", common.K8sLabelSource),
		},
		Trace: policy.TRACE_VERBOSE,
	}
	c.Assert(node.Allows(&ctx), Equals, api.DENY)
```
I will change the `coverAll` behavior for my point of view. Makes sense?